src/scalper.ts

```diff
@@ -355,4 +360,13 @@ function getDIPGamma(dipProduct: DIP[], fairValue: number){
   return gammaSum
 }
 
+function timeSinceMidDay(){
+  const timeNow = new Date();
+  const year = timeNow.getUTCFullYear();
+  const month = timeNow.getUTCMonth();
+  const day = timeNow.getUTCDate();
+  const timeCheckUTC = Date.UTC(year, month, day, 12, 0 ,0 ,0);
```

Comment:
it might be easier to do it this way
https://github.com/Dual-Finance/dual-labs-dip/blob/master/app/src/pages/pricing/Pricing.tsx#L30

Reply:
good to just get a fresh run everyday at 12 UTC rather than just expo days